doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -462,6 +462,17 @@ Displaying information about notations
 
    :flag:`Printing All` to disable other elements in addition to notations.
 
+.. cmd:: Print Notation @string
+
+   Displays the level and associativity of a notation given by a parsing rule
+   :token:`string`.
+
+   .. example::
+
+      .. coqtop:: all
+
+         Reserved Notation "x $ y" (at level 123, right associativity).
+         Print Notation "x $ y".
```

Comment:
![image](https://user-images.githubusercontent.com/1253341/155425201-a64afb9e-1d79-4a3f-8a77-5b817a73b0db.png)

How about showing the info in the order it appears in the `Reserved Notation` and include the constant `$`?  Ex:
`"(constr NEXT LEVEL) $ (constr LEVEL 123)" at LEVEL 123, right associativity`?  I used caps to make the keywords stand out.  Not sure if that's workable for all cases.


You have an extra space before "right"

Reply:
I'm not sure how to print the information in that form. Right now we are just reusing the printers used in error messages.

> You have an extra space before "right"

Thanks, I'm fixing that now.